### ISSUE TITLE
Spectral image viewer: synchornized scrolling

### ISSUE THREAD

**Issue by awisnicki** [livingstoneonline/livingstoneonline#94]
The synchronized scrolling only works to a degree, but the images can also be independently manipulated. The image should always be linked so that whatever you do to one also happens to the other.

**Comment by nigelgbanks** [comment]
Ready for review on stage.

**Comment by awisnicki** [comment]
This works well, so thank you. However, I do have a follow up question.

When I look at two images and zoom in, then when I switch one of the images it resets to both images zoomed out (default view). Would it be possible to have it work as follows: 

1) Zoom in on both images, 
2) Select new image in one pane, 
3) Current zoom remains and new image shows up zoomed into the right place.

Hope that makes sense. 

In other words, for instance, I'd like to examine a small area on an image. I zoom in and then might want to look at that area in multiple images so would be switching between different images, so I wouldn't want to zoom in again each time.

**Comment by nigelgbanks** [comment]
Ready to review on stage.

**Comment by awisnicki** [comment]
This works brilliantly. Well done!
